data/json/monsters/zed_lieutenant.json

```diff
@@ -0,0 +1,239 @@
+          "t_forestfloor",
+          "t_moss",
+          "t_underbrush",
+          "t_underbrush_harvested_spring",
+          "t_underbrush_harvested_summer",
+          "t_underbrush_harvested_autumn",
+          "t_underbrush_harvested_winter",
+          "t_tree_young"
+        ]
+      }
+    ]
+  },
+  {
+    "type": "ter_furn_transform",
+    "id": "ltnt_tree_drain",
+    "//": "TODO: Add all the tree types! All of them! Including the harvested ones! ;_;",
+    "terrain": [ { "result": [ "t_tree_very_dead" ], "valid_terrain": [ "t_tree" ] } ]
+  },
+  {
+    "name": "GROUP_AMALGAMATIONS_SPAWNED",
+    "type": "monstergroup",
+    "//": "TODO: Borrow more of Venera's fun little twisted creations to explain why biomass = instant abberations.",
+    "monsters": [ { "monster": "mon_amalgamation_swarmer", "weight": 3, "pack_size": [ 4, 6 ] } ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_LIEUTENANT_ACTIVATE_SHADOW",
+    "recurrence": 1,
+    "//1": "TODO: 6 months into the cataclysm AND # of zombie kills?",
+    "condition": { "and": [ { "days_since_cataclysm": 180 }, { "math": [ "u_Lieutenants_active", "<", "1" ] } ] },
+    "//2": "Integer value instead of bool, so more can be added later and use math for conditions",
+    "effect": [ { "u_add_var": "Lieutenants_active", "value": "1" }, { "u_add_var": "Shadow_Lurking", "value": "1" } ],
+    "deactivate_condition": { "math": [ "u_Lieutenants_active", ">=", "1" ] }
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_LIEUTENANT_SPAWN_SHADOW",
+    "//": "TODO: Check every once in a while but not instantly, make sure he's not an every-night sort of thing. Should be about 1 in 5 nights, and can be instantly dismissed by simply getting to safety(EOC_LIEUTENANT_SHADOW_WITHDRAWS_FOR_NOW)",
+    "recurrence": 1,
+    "//1": "Night time, shadow is active, you are outdoors, and you are not likely to be near shelter.",
+    "condition": {
+      "and": [
+        { "not": "is_day" },
+        { "math": [ "u_Lieutenants_active", ">", "0" ] },
+        { "math": [ "u_Shadow_Lurking", ">", "0" ] },
+        "u_is_outside",
+        {
+          "or": [ { "u_at_om_location": "field" }, { "u_at_om_location": "forest" }, { "u_at_om_location": "forest_thick" } ]
+        }
+      ]
+    },
+    "effect": [
+      { "u_adjust_var": "Shadow_Lurking", "adjustment": -1 },
+      { "u_spawn_monster": "mon_lieutenant_shadow", "real_count": 1, "min_radius": 15, "max_radius": 20 },
+      { "u_message": "You shiver as the night closes in...", "type": "bad", "popup": true }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_LIEUTENANT_SHADOW_WITHDRAWS_FOR_NOW",
+    "recurrence": 5,
+    "//1": "Lots of friends nearby, or in base camp, or safely indoors, or the sun has risen. TODO: is_day evaluates true before sundeath kicks in?",
+    "condition": { "or": [ { "npc_allies": 3 }, { "u_at_om_location": "FACTION_CAMP_ANY" }, { "not": "u_is_outside" }, "is_day" ] },
```

Comment:
I think your operator here might be wrong as you want greater or equal 3.

Reply:
The docs *say* that npc_allies returns true if at least that number of allies is in the reality bubble, no operator needed.

I'll put it down for checking though.

Reply:
(It does in fact work as advertised)